Podfile

```diff
@@ -65,3 +65,8 @@ target 'AzureStorageBlobDemo' do
   project 'examples/AzureStorageBlobDemo/AzureStorageBlobDemo'
   pod 'MSAL', '~> 1.1.2'
 end
+
+target 'AzureSDKDemoSwiftUI' do
+  project 'examples/AzureSDKDemoSwiftUI/AzureSDKDemoSwiftUI'
+  pod 'MSAL', '~> 1.1.2'
```

Comment:
We can remove the MSAL dependency for this. When/if MSAL is Swift-UI friendly, we can reintroduce it.

Reply:
So it turns out I actually need MSAL. 
https://github.com/Azure/azure-sdk-for-ios/pull/675/files#diff-8abc0fc0a497bc17bcf95e2e95626b721c7f3d3f52b7a670dcb1fb53e6333839R86
I need it to actually create the blob storage client

Reply:
You can create the client using the `StorageBlobClient.init(connectionString:withOptions:)` initializer instead. That's why it has to be read-only connection string.

Reply:
I updated the code to use init from connection string but at run time looks like the `AzureStorageBlob` internally has a dependency on it. 
```
dyld: Library not loaded: @rpath/MSAL.framework/MSAL
  Referenced from: .../Developer/Xcode/DerivedData/AzureSDK-dwmootmmaabdspgmqqrzmsanobci/Build/Products/Debug-iphonesimulator/AzureStorageBlob.framework/AzureStorageBlob
  Reason: image not found
  ```

Reply:
Should only be this: 
![image](https://user-images.githubusercontent.com/5723682/108122346-fc76a900-7058-11eb-8f2a-1e15a3ef95c2.png)

However, I'm okay leaving it in and I can take a look later, since we do want the Storage library to be usable without MSAL.

Reply:
I have removed the MSAL implementation from the rest of the code.   But for the time being I am leaving this in the podfile.